pulsar/json_schemas/channel.json

```
@@ -0,0 +1,87 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "http://asyncapi.com/bindings/pulsar/channel.json",
+    "title": "Channel Schema",
+    "description": "This object contains information about the channel representation in Pulsar, which covers namespace and topic level admin configuration. This object contains additional information not possible to represent within the core AsyncAPI specification.",
+    "type": "object",
+    "additionalProperties": false,
+    "patternProperties": {
+      "^x-[\\w\\d\\.\\-\\_]+$": {
+        "$ref": "https://raw.githubusercontent.com/asyncapi/spec-json-schemas/v2.14.0/schemas/2.4.0.json#/definitions/specificationExtension"
+      }
+    },
+    "properties": {
+      "namespace": {
+        "type": "string",
+        "description": "The namespace, the channel is associated with. If omitted, 'public' MUST be assumed."
+      },
+      "persistence": {
+        "type": "string",
+        "enum": [
+          "persistent",
+          "non-persistent"
+        ],
+        "description": "persistence of the topic in Pulsar."
+      },
+      "compaction": {
+        "type": "integer",
+        "minimum": 0,
+        "description": "Topic compaction threshold given in MB"
+      },
+      "geo-replication": {
+        "type": "array",
+        "description": "A list of clusters the topic is replicated to."
```

You can go even further and define it's an array of strings:

```suggestion
        "description": "A list of clusters the topic is replicated to.",
        "items": {
          "type": "string"
        }
```

---

Awesome, thanks a lot for all of the edits, ill move things around as suggested as well.